data/en/entityloadbypk.json

```diff
@@ -1,18 +1,18 @@
 {
 	"name":"entityLoadByPK",
 	"type":"function",
-	"syntax":"entityLoadByPK(entityName, id)",
+	"syntax":"entityLoadByPK(entity, id)",
 	"returns":"any",
 	"related":["entityLoadByExample"],
-	"description":" Loads and returns an array of objects for given primary key.",
+	"description":"Loads and returns an array of objects for given primary key.\nUse this function to avoid specifying the required boolean parameter in `EntityLoad()`.",
 	"params": [
-		{"name":"entityName","description":"Name of the entity to be loaded","required":true,"default":"","type":"string","values":[]},
-		{"name":"id","description":"ID of the entity to be loaded","required":true,"default":"","type":"string","values":[]},
-		{"name":"unique","description":"Lucee4.5+ Specify whether you expect multiple or just one record\nIf set to `true`, the entity is returned\nIf set to `false` an array is returned to support multiple entities\nIf set to `true` but multiple entities would be returned an error will be thrown","required":false,"default":"","type":"boolean","values":[]}
+		{"name":"entity","description":"Name of the entity to be loaded.","required":true,"default":"","type":"string","values":[]},
+		{"name":"id","description":"ID of the entity to be loaded.","required":true,"default":"","type":"string","values":[]},
+		{"name":"unique","description":"LUCEE 4.5+ : Specify whether you expect multiple or just one record.\nIf set to `true`, the entity is returned.\nIf set to `false` an array is returned to support multiple entities.\nIf set to `true` but multiple entities would be returned an error will be thrown.","required":false,"default":"","type":"boolean","values":[]}
```

Comment:
I was basically saying to revert the change because that is the way the tag works. Look at how it's styled already in [CFDocs](https://cfdocs.org/entityloadbypk).

Reply:
Gotcha. Fixed properly this time.